Source/JavaScriptCore/b3/air/AirCustom.cpp

```diff
@@ -64,7 +65,20 @@ bool CCallCustom::isValidForm(Inst& inst)
     if (!value)
         return false;
 
-    if (inst.args.size() != (value->type() == Void ? 0 : 1) + value->numChildren())
+    size_t resultCount = cCallResultCount(value);
+    size_t expectedArgCount = resultCount;
+    for (Value* child : value->children()) {
+        ASSERT_UNUSED(child, child->type() != Tuple);
+#if USE(JSVALUE32_64)
+        if (child->type() == Int64) {
+            expectedArgCount += 2;
```

Comment:
We have similar logic to this elsewhere. Do we just want a function called something like "numberOfRegistersForInt64"?

Reply:
+1